book/06-github/sections/3-maintaining.asc

```diff
@@ -15,102 +15,102 @@ image::images/newrepo.png[The ``Your repositories'' area.]
 
-Pull Requests can either come from a branch in a fork of your repository or they can come from another branch in the same repository. The only difference is that the ones in a fork are often from people where you can't push to their branch and they can't push to yours, whereas with internal Pull Requests generally both parties can access the branch.
+Pull Request bisa berasal dari branch di fork repositori Anda atau bisa berasal dari branch lain di repositori yang sama. Perbedaannya adalah bahwa branch di fork sering kali berasal dari orang-orang yang tidak dapat Anda melakukan push ke branch mereka dan mereka tidak dapat melakukan push ke branch Anda, sedangkan dengan Pull Request Internal umumnya, kedua belah pihak dapat mengakses branch tersebut.
 
-For these examples, let's assume you are ``tonychacon'' and you've created a new Arudino code project named ``fade''.
+Untuk contoh ini, mari kita anggap Anda adalah `tonychacon'' dan Anda telah membuat proyek kode Arudino baru yang bernama ``fade''.
 
 [[_email_notifications]]
 ===== Email Notifications
 
-Someone comes along and makes a change to your code and sends you a Pull Request. You should get an email notifying you about the new Pull Request and it should look something like <<_email_pr>>.
+Seseorang datang dan membuat perubahan pada kode Anda dan mengirimkan Pull Request kepada Anda. Anda pasti mendapatkan surel yang memberi tahu Anda tentang Pull Request yang baru dan seharusnya terlihat seperti <<_email_pr>>.
 
 [[_email_pr]]
 .Email notification of a new Pull Request.
 image::images/maint-01-email.png[Pull Request email notification]
 
-There are a few things to notice about this email. It will give you a small diffstat -- a list of files that have changed in the Pull Request and by how much. It gives you a link to the Pull Request on GitHub. It also gives you a few URLs that you can use from the command line.
+Ada beberapa hal yang harus diperhatikan mengenai surel ini. Ini akan memberi Anda diffstat kecil -- daftar berkas yang telah berubah dalam Pull Request dan seberapa banyak. Ini memberi Anda tautan ke Pull Request pada GitHub. Ini juga memberikan Anda beberapa URL yang dapat Anda gunakan dari baris perintah.
 
-If you notice the line that says `git pull <url> patch-1`, this is a simple way to merge in a remote branch without having to add a remote. We went over this quickly in <<_checking_out_remotes>>. If you wish, you can create and switch to a topic branch and then run this command to merge in the Pull Request changes.
+Jika Anda melihat baris yang mengatakan `git pull <url> patch-1`, ini adalah cara mudah untuk melakukan merge branch remote tanpa harus menambahkan remote. Kami membahasnya dengan cepat di <<_checking_out_remotes>>. Jika Anda mau, Anda dapat membuat dan beralih ke branch topik dan kemudian menjalankan perintah ini untuk di-merge-kan dalam perubahan Pull Request.
```

Comment:
We di sini adalah kita. Kita mengajak pembaca melakukan apa yang akan dilakukan.